### PR TITLE
Polish

### PR DESCRIPTION
**Pull request by KSXGitHub**

### CODE REVIEW

File: tests/test_main.rs [hunk 1]
```diff
@@ -1,5 +1,5 @@
-use assert_cmd::prelude::*;
-use build_fs_tree::*;
+use assert_cmd::prelude::{CommandCargoExt, OutputAssertExt};
+use build_fs_tree::{dir, file, Build as _, MergeableFileSystemTree};
```
Comment (KSXGitHub) [inline tests/test_main.rs:2]:
```suggestion
use build_fs_tree::{dir, file, Build, MergeableFileSystemTree};
```

**Review by zkochan — APPROVED** [review]

**Comment by KSXGitHub** [comment]
@zkochan I just added even more commits.

**Review by zkochan — APPROVED** [review]

**Comment by KSXGitHub** [comment]
This should be my final commits. If you are satisfied, you may merge this to `workspace-root` and then to `main`.

**Comment by zkochan** [comment]
I thought the convention was to include scope with function from other mods

**Comment by KSXGitHub** [comment]
> I thought the convention was to include scope with function from other mods

What are you referring to?

**Comment by zkochan** [comment]
```
    -let temp_dir = tempfile::tempdir().unwrap();
    +let temp_dir = tempdir().unwrap();
```

I was watching a tutorial, they said to use scope for functions from different modules.

**Comment by KSXGitHub** [comment]
It's just opinions really. You would have to use `tmpfile::` if there are conflicting names with `tempdir`, that's all. There is no change in runtime behavior.

**Comment by zkochan** [comment]
I understand that it doesn't affect runtime. The presenter said it is good because you know the function is from a different file. Anyway, I don't care about it right now.

**Comment by KSXGitHub** [comment]
BTW, I will mention it twice in case you missed it:

To install Clippy and Rustfmt to the CLI, run:

```sh
rustup toolchain install stable
rustup component add clippy rustfmt
```

To run clippy:

```sh
cargo clippy
```

Run run rustfmt:

```sh
cargo fmt
```

**Pull request merged**